connector_elasticsearch/models/se_index_config.py

```diff
@@ -18,6 +18,16 @@ class SeIndexConfig(models.Model):
         compute="_compute_body_str", inverse="_inverse_body_str"
     )
 
+    @api.model
+    def create(self, values):
+        # When creating index from the form, inverse function is not called
+        # and 'body' will be empty, which leads to validation error. So set
+        # 'body' when creating the record if 'body_str' is not empty.
+        if 'body' not in values and 'body_str' in values:
```

Comment:
@ChinaShrimp Thank you for your patch and welcome on the OCA/search-engine repo :smirk:
I'm a little bit astonished. Inverse method should always be called. Can you describe the steps to reproduce the problem?
Regards,

Reply:
@lmignon I'm using Odoo Community 12.0:
1. reset to a clean database;
2. install elasticsearch connector;
3. create a se index config;
4. fill name and copy a valid json to body_str;
5. save
6. got validation error (body is empty)

I add print log in inverse function and be really astonished too. In Odoo 12.0 (I didn't try other versions), for new record creation, it will create the record first and then inverse method is called.

I digged a little bit and found there were some issue discussions regarding that : 
https://www.google.com/search?q=inverse+method+odoo+not+called&oq=inverse+method+odoo+not+called&aqs=chrome..69i57j0.7001j0j7&sourceid=chrome&ie=UTF-8

Thanks,
Lyon

Reply:
@Lyon thank you for the debug.
Therefore your fix seems to be the right one. Could you adaptyour comment into the create method to specify that it's required to manually call the inverse method in case of creation since the method is called after the insert into database and the field body is required...

Reply:
@lmignon Thanks for your suggestion, I have reworked the comment based on your suggestion and test it using Odoo 12.0 community.